cli/src/main/java/com/liferay/blade/cli/util/BladeUtil.java

```diff
@@ -90,7 +90,10 @@ public static void downloadLink(String link, Path target) throws IOException {
 		}
 	}
 
-	public static File findParentFile(File dir, String[] fileNames, boolean checkParents) {
+	@SafeVarargs
+	public static File findParentFile(
+		File dir, String[] fileNames, boolean checkParents, Predicate<File>... predicates) {
```

Comment:
THere are no callers that use the `predicates` argument.  Can you remove it?  THanks.